syntax = "proto3";

package filetransfer;

service FileTransfer {
  rpc SendFile(FileRequest) returns (FileResponse) {};
  rpc ReceiveFile (FileRequest) returns (FileResponse);
}

message FileRequest {
  string filename = 1;
  bytes content = 2;
}

message FileResponse {
  string message = 1;
  bytes content = 2;
}
